### DISCUSSION TITLE
reordenar wiki

### DISCUSSION THREAD

**Discussion by claubott** [rpgrca/proyecto-ispc-fullstack#242] [General]
Hola! 
En el menú de la wiki, las páginas se ordenan solas por orden alfabético. Así está ahora:

Home
Análisis y diseño del sistema
Controladores del sistema
Diagrama de clases
Diagrama de colaboración entre clases
Diagrama entidad relación
Diagrama relacional
Especificación de requisitos de software
Manual interno de procedimientos
Sección 1: Introducción
Sección 2: Descripción general
Sección 3: Requisitos específicos
Sitio Institucional
Vista del sistema

Home siempre es la primera página de la wiki. Pero se puede hacer un **custom sidebar** para tener una tabla de contenido con otro orden, e Incluso se pueden agrupar varias páginas bajo algún título. 

Luego del Home, originariamente teníamos: Especificación de requisitos de software y las secciones 1, 2 y 3. 

**Cómo se podrían agrupar los demás temas/páginas?**

**Comment by rpgrca** [comment]
El home tendría el orden más o menos correcto, la parte de análisis con la documentación del ERS con sus tres secciones, después la parte de diseño con el mapa de sitio y todos los diagramas, además de los controladores, y después la parte sobre el sitio institucional que es un proyecto aparte. Al final el manual interno que digamos que es un manual de referencia nuestro y no necesariamente algo perteneciente al proyecto, podría considerarse un anexo.
